simulariumio/constants.py

```diff
@@ -24,7 +24,7 @@ class V1_SPATIAL_BUFFER_STRUCT:
     R_INDEX: int = 9  # radius
     NSP_INDEX: int = 10  # num subpoints
     SP_INDEX: int = 11  # subpoints
```

Comment:
change "subpoints" to "index of first subpoint, only valid if num_subpoints > 0"